skpro/regression/multiquantile.py

```diff
@@ -0,0 +1,362 @@
+                "or alpha provided when instantiated"
+            )
+
+        # check that we have regressors for alpha
+        if not np.in1d(alpha, self.alpha).any():
+            raise ValueError(
+                "No regressor(s) for provided alpha. Fitted alpha: "
+                f"{self.alpha}. Provided alpha: {alpha}."
+            )
+
+        # predict
+        def _predict_quantile(X, regressor):
+            pred = regressor.predict(X=X)
+            pred = np.array(pred).flatten()
+            return pred
+
+        X_pred = np.array(X)
+        regressors = [self.regressors_[a] for a in alpha]
+        preds = Parallel(n_jobs=self.n_jobs)(
+            delayed(_predict_quantile)(X_pred, regressor) for regressor in regressors
+        )
+
+        # format predictions as DataFrame
+        preds = pd.DataFrame(np.transpose(preds))
+        preds.columns = pd.MultiIndex.from_product([[self._y_varname], alpha])
+        if isinstance(X, (pd.DataFrame, pd.Series)):
+            preds.index = X.index
+
+        # solve quantile crossing problem
+        if self.sort_quantiles:
+            preds = preds.apply(
+                lambda row: row.sort_values().values, axis=1, result_type="broadcast"
+            )
+
+        return preds
+
+    def get_params(self, deep=True):
+        """Get a dict of parameters values for this object.
+
+        Parameters
+        ----------
+        deep : bool, default=True
+            Whether to return parameters of components.
+
+            * If True, will return a dict of parameter name : value for this object,
+              including parameters of components (= BaseObject-valued parameters).
+            * If False, will return a dict of parameter name : value for this object,
+              but not include parameters of components.
+
+        Returns
+        -------
+        params : dict with str-valued keys
+            Dictionary of parameters, paramname : paramvalue
+            keys-value pairs include:
+
+            * always: all parameters of this object, as via `get_param_names`
+              values are parameter value for that key, of this object
+              values are always identical to values passed at construction
+            * if `deep=True`, also contains keys/value pairs of component parameters
+              parameters of components are indexed as `[componentname]__[paramname]`
+              all parameters of `componentname` appear as `paramname` with its value
+            * if `deep=True`, also contains arbitrary levels of component recursion,
+              e.g., `[componentname]__[componentcomponentname]__[paramname]`, etc
+        """
+        params = super().get_params(deep=deep)
+        # remove quantile_regressor_alpha_param from params
+        # managed by self, so removal prevents user confusion
+        params.pop(f"quantile_regressor__{self.quantile_regressor_alpha_param}", None)
```

Comment:
I don't think we want that? Or, do you want to use different regressors per alpha? That would work differently (let me know if that is what you want to build)